superset/examples/countries.py

```diff
@@ -2498,13 +2498,8 @@
         all_lookups[lookup][country[lookup].lower()] = country
 
 
-def get(field, symbol):
+def get(field: str, symbol: str) -> Optional[Dict[str, Any]]:
     """
     Get country data based on a standard code and a symbol
-
```

Comment:
These examples were clearly wrong given the return type and thus I removed them.